docs/content/querying/limitspec.md

```diff
@@ -13,9 +13,19 @@ The default limit spec takes a limit and the list of columns to do an orderBy op
     "type"    : "default",
     "limit"   : <integer_value>,
     "columns" : [list of OrderByColumnSpec],
+    "limitGranularity" : [QueryGranularity],
+    "limitPerGroup: <boolean_value>
 }
 ```
 
+|property|description|required?|
+|--------|-----------|---------|
+|type|This String should always be "default". |yes|
+|limit|Number of limit.|yes|
+|columns|Ordering to apply limit; see [OrderByColumnSpec]. |no|
+|limitGranularity|Granularity for limit. see [Granularities](../querying/granularities.html).|no|
+|limitPerGroup|Whether to apply limit to each group; User can possibly get up to <limit> * <number of groups> rows. |no|
```

Comment:
Oh I see, `regroup=all` is what I would want. In fact I would issue that on queries by default. I could see `regroup=null` and `regroup=2d` being useful in some edge cases. I would make a point of advertising `regroup=all` in the docs as IMO it is by far the most useful configuration.